pkg/agent/openflow/client.go

```diff
@@ -294,6 +294,25 @@ type Client interface {
 	UninstallTrafficControlReturnPortFlow(returnOFPort uint32) error
 
 	InstallMulticastGroup(ofGroupID binding.GroupIDType, localReceivers []uint32) error
+
+	// InstallMulticlusterNodeFlows installs flows to classify cross-cluster packets from a general Node to a Gateway.
+	InstallMulticlusterNodeFlows(
+		hostname string,
+		peerConfigs map[*net.IPNet]net.IP,
+		tunnelPeerIP net.IP) error
+
+	// InstallMulticlusterGatewayFlows installs flows to classify cross-cluster packets from a local Gateway to a remote Gateway.
+	InstallMulticlusterGatewayFlows(
+		hostname string,
+		peerConfigs map[*net.IPNet]net.IP,
+		tunnelPeerIP net.IP,
+		localGatewayIP net.IP) error
+
+	// InstallMCClassifyFlows installs flows to classify cross-cluster packets.
+	InstallMCClassifyFlows(hostname string, tunnelOFPort uint32, isGateway bool) error
```

Comment:
InstallMulticastClassifierFlows if it is about packet classification (like tunnelClassifierFlow, etc.).

Reply:
done